src/components/CommunityModuleBadge.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {StyleSheet, View, Text} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import {SymbolIcon} from 'react-native-xaml';
+
+const createStyles = (colors: any) =>
+  StyleSheet.create({
+    badgeContainer: {
+      height: 28,
+      width: 190,
+      borderRadius: 2,
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: colors.text,
+      margin: 2,
+    },
+    badgeText: {
+      fontSize: 15,
+      fontWeight: '400',
+      color: 'white',
+      paddingRight: 4,
+    },
+  });
+
+export function CommunityModuleBadge() {
```

Comment:
W have multiple (3?) badges with a very similar style/layout. Is there any reason why we couldn't have just one base component with props for the symbol/text?

Reply:
Sure I can make that logic change!